### PR TITLE
OSDOCS-3097: overview

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.7 and 4.6 (  cherry pick to 4.6 as both versions have the same extra module )
- [Jira](https://issues.redhat.com/browse/OSDOCS-3097)
- [Preview](https://deploy-preview-40170--osdocs.netlify.app/openshift-enterprise/latest/monitoring/understanding-the-monitoring-stack)

### CODE REVIEW

File: monitoring/index.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+The growth of containerization has not only brought many advantages to developers,DevOps, and IT teams but also challenges. The flexibility and scalability of containerized applications pose challenges since there is no direct relationship between an application and the server it runs on.
```
Comment (juzhao) [inline monitoring/index.adoc:8]: `developers, `
should add a space after `,`

File: monitoring/index.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+The monitoring stack is available by default, when {product-title} is installed. The cluster administrator has the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects].
+You can also xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations.
+You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
+* xref:../monitoring/managing-alerts.adoc#managing-alerts[Manage alerts]
+* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Review monitoring dashboards]
+
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics,alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI.
```
Comment (juzhao) [inline monitoring/index.adoc:22]: `metrics, ` should add a space after `,`

File: monitoring/index.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+You can also xref:../monitoring/exposing-custom-application-metrics-for-autoscaling.adoc#exposing-custom-application-metrics-for-autoscaling[expose custom application metrics] for horizontal pod autoscaling.
+
+As a cluster administrator you can determine why:
+
+* xref:../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[User-metrics are not available]
```
Comment (juzhao) [inline monitoring/index.adoc:27]: It is better change the link to
`../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues#investigating-why-user-defined-metrics-are-unavailable_troubleshooting-monitoring-issues`

File: monitoring/index.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* xref:../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[Prometheus consumes a lot of disk space]
```
Comment (juzhao) [inline monitoring/index.adoc:28]: It is better change the link to
`../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues#determining-why-prometheus-is-consuming-disk-space_troubleshooting-monitoring-issues`

**Comment by juzhao** [comment]
> * Applies to 4.7 only ( but can this be cherry picked to 4.6 as both versions have the same extra module? )
we can cherry pick to 4.6

File: monitoring/index.adoc (hunk 4 above)
Comment (Amrita42) [reply monitoring/index.adoc:28]: tooling limitations we cant xref to a module.

**Pull request closed without merging**